### PR TITLE
update CONTRIBUTING, fix bash and mysql in makefile

### PR DESCRIPTION
**Pull request by alanb2718**
Could one or both of you have a look at this please?  There are some TODO items remaining to be fixed before it's ready for the main branch; plus it should be checked over in any case.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -40,18 +39,46 @@ DB_HOST=db
...
+
+Start the root server using `make dev` (see above).  Then in a separate terminal, run the tests using `make test`.
+Somewhat annoyingly, `make test` will clobber your current database, so you'll need to restore it if you want to go
+back to running the root server.
+
+
+## Running lint
+You can run the linter by running `make lint` in the top-level directory.
+
+TODO: this didn't work for me
```
Comment (jbraswell) [inline CONTRIBUTING.md:67]: It doesn't work when xdebug is listening... maybe that was the problem.

File: Makefile [hunk 2]
```diff
@@ -102,11 +102,11 @@ docker-publish-base:  ## Builds Base Docker Image
 
 .PHONY: mysql
 mysql:  ## Runs mysql cli in mysql container
-	docker exec -it docker_db_1 mysql -u root -prootserver rootserver
+	docker exec -it docker-db-1 mysql -u root -prootserver rootserver
```
Comment (jbraswell) [inline Makefile:105]: Curiously, this doesn't work on my machine. It is underscored for me.

**Review by jbraswell — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (jbraswell) [reply Makefile:105]: I'll enable v2, ignore this comment.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -40,18 +39,46 @@ DB_HOST=db
 DB_PREFIX=na
 ```
 
+## Useful `make` commands
```
Comment (jbraswell) [inline CONTRIBUTING.md:42]: `make help` is also useful

```
[~/git/bmlt-enabled/bmlt-root-server]$ make help
bash                           Runs bash shell in apache container
clean                          Clean build
composer                       Runs composer install
coverage-serve                 Serves HTML Coverage Report
coverage                       Generates HTML Coverage Report
crouton                        Installs crouton
dev                            Docker Compose Up
docker-publish-base            Builds Base Docker Image
docker-push                    Pushes docker image to Dockerhub
docker                         Builds Docker Image
help                           Print the help documentation
lint-fix                       PHP Lint Fix
lint                           PHP Lint
mysql                          Runs mysql cli in mysql container
test                           Runs PHP Tests
zip                            Builds zip file
[~/git/bmlt-enabled/bmlt-root-server]$ 

```

Comment (jbraswell) [reply CONTRIBUTING.md:42]: The `make clean` target really removes all build artifacts and downloaded dependencies. Not sure how to say that succinctly.

You really only need to do `make docker` when first getting set up and after a change has been made to the Dockerfile or its base image.

File: CONTRIBUTING.md (hunk 1 above)
Comment (alanb2718) [reply CONTRIBUTING.md:67]: Yes, that was it.

File: CONTRIBUTING.md (hunk 3 above)
Comment (alanb2718) [reply CONTRIBUTING.md]: Added these.

**Review by jbraswell — APPROVED** [review]

**Pull request merged**